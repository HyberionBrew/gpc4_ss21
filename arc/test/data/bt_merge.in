1: x = 6
2: y = 5
3: x = 4
4: x = 2
4: y = 7