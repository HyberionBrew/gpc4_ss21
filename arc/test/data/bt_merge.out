1: z = 6
2: z = 5
3: z = 4
4: z = 2
